docs/CHANGELOG.asciidoc

```diff
@@ -77,6 +78,352 @@ Review important information about the {kib} 8.x releases.
 
 include::upgrade-notes.asciidoc[]
 
+
+[[release-notes-8.16.0]]
+== {kib} 8.16.0
+
+For information about the {kib} 8.16.0 release, review the following information.
+
+[float]
+[[deprecations-8.16.0]]
+=== Deprecations
+
+The following functionality is deprecated in 8.16.0, and will be removed in 9.0.0.
+Deprecated functionality does not have an immediate impact on your application, but we strongly recommend
+you make the necessary updates after you upgrade to 8.16.0.
+
+[discrete]
+* The Logs Stream is now hidden by default in favor of the Logs Explorer app.
+[%collapsible]
+====
+*Details* +
+You can find the Logs Explorer app in the navigation menu under Logs > Explorer, or as a separate tab in Discover. For more information, refer to ({kibana-pull}194519[#194519]).
+
+*Impact* +
+You can still show the Logs Stream app again by navigating to Stack Management > Advanced Settings and by enabling the `observability:enableLogsStream` setting.
+====
+      
+[discrete]
+* Deprecates the Observability AI Assistant specific advanced setting `observability:aiAssistantLogsIndexPattern`.
+[%collapsible]
+====
+*Details* +
+The Observability AI Assistant specific advanced setting for Logs index patterns `observability:aiAssistantLogsIndexPattern` is deprecated and no longer used. The AI Assistant will now use the existing **Log sources** setting `observability:logSources` instead. For more information, refer to ({kibana-pull}192003[#192003]).
+
+//*Impact* +
+//!!TODO!!
+====
+    
+
+      
+[float]
+[[features-8.16.0]]
+=== Features
+{kib} 8.16.0 adds the following new and notable features.
+
+AGPL license::
+* Adds AGPL 3.0 license ({kibana-pull}192025[#192025]).
+Alerting::
+* Support TheHive connector in cases ({kibana-pull}180931[#180931]).
+* Adds TheHive connector ({kibana-pull}180138[#180138]).
```

Comment:
I think Cases deserves to have its section on every release so users can see Cases related features easier. Alerting is not intuitive if I want to see the new Cases features.

```
Alerting::
* A new connector for TheHive ({kibana-pull}180138[#180138]).

Cases::
* Support of TheHive connector ({kibana-pull}180931[#180931]).
```

Reply:
The trick here is that the cases PR had both the connector and case feature labels, so the release note generator has to pick only one. We can override manually for now as necessary and hopefully have a clearer process as we re-evaluate the release notes process in the new system.

Reply:
Done via b20d638e7ddf062aece7999254002bae5b12d984